configurations/performance/cassandra_stress_gradual_load_steps_enterprise.yaml

```diff
@@ -0,0 +1,4 @@
+# Define load ops for steps
+perf_max_ops: {"read": 794000, "write": 405000, "mixed": 530000}
+perf_threads: {"read": 620, "write": 700, "mixed": 1900}
+perf_gradual_throttle_steps: {"read": '150000', '300000', '450000', '600000', '700000', 'unthrotteled', "mixed": ['50000', '150000', '300000', '450000', 'unthrotteled']}  # where every value is in ops
```

Comment:
"write": '200000', '300000', '400000', 'unthrotteled'

Not sure why the format is different between read and mixed.